documentation/docs/03-loading.md

```diff
@@ -92,7 +92,7 @@ The `load` function receives an object containing four fields — `page`, `fetch
 
 #### page
 
-`page` is a `{ host, path, params, query }` object where `host` is the URL's host, `path` is its pathname, `params` is derived from `path` and the route filename, and `query` is an instance of [`URLSearchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams).
+`page` is a `{ host, path, params, query }` object where `host` is the URL's host, `path` is its pathname, `params` is derived from `path` and the route filename, and `query` is an instance of [`URLSearchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams). Mutating `page` does not update the current URL, rather, it should be done by means of navigation using `goto`.
```

Comment:
@benmccann @bluwy can we add this information here?

Reply:
this sounds reasonable. I wonder if an even better solution than documenting it might be to make the store exposed to the user readonly? https://github.com/sveltejs/svelte/pull/6518

Reply:
I think `page` is referred here as an object rather than a store, but if it was imported from `$app/stores`, it's already a [readable store](https://kit.svelte.dev/docs#modules-$app-stores). So we would need to freeze the object, but I'm not sure if it's something we want to do. The change looks good to me though.

Reply:
```suggestion
`page` is a `{ host, path, params, query }` object where `host` is the URL's host, `path` is its pathname, `params` is derived from `path` and the route filename, and `query` is an instance of [`URLSearchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams). Mutating `page` does not update the current URL; you should instead navigate using [`goto`](#modules-$app-navigation).
```